docs/presentations/DB-write-speed/db-write-speed.slide

```diff
@@ -0,0 +1,501 @@
+# "RDS burst balances are depleted"
+DB write speed in External data pipeline database
+28 Mar 2022
+Tags: golang, go, rds, database, burst balance
+
+Pavel Tišnovský
+Red Hat, Inc.
+<ptisnovs@redhat.com>
+
+
+
+## Problem description
+
+- Alerts: Insights databases have depleted RDS burst balances: ccx-notification-prod, ...
+- Consuming & writting results into DB is **slow** after service outage
+
+
+
+## Kafka lag on production after service outage
+
+- example from real outage that lasted just 30 minutes
+- 5 hours(!) to fully recover
+
+.image kafka1.png _ 1020
+
+
+
+## Burst mode versus "economy" mode
+
+- burst mode supported by AWS (so called gp2 volumes)
+- but it is not enough
+
+.image kafka2.png _ 1020
+
+
+
+## Burst mode
+
+- gp2 volumes
+- 5.4 million I/O credits at 3000 IOPS
+- 3000 IOPS for 30 minutes
+- once the burst is exhausted, it delivers only 100 IOPS
+- replenished at the rate 3 IOPS per GiB per second
+- during downtime, burst credit can be replenished
+- Burst/baseline performances
+    - burst performance is constant
+    - baseline performance increases linearly with volume size
+    - for volumes higher that 1TB, burst performance is irrelevant
```

Comment:
"for volumes higher tha**n** 1TB, burst performance is irrelevant"

Reply:
ah good catch!